jicofo-common/src/main/kotlin/org/jitsi/jicofo/xmpp/muc/ChatRoomImpl.kt

```diff
@@ -400,7 +410,7 @@ class ChatRoomImpl(
         // Unblock any threads waiting on the latches
         roomMetadataLatch.countDown()
         roomJoinedLatch.countDown()
-        presenceQueue.close()
+        xmppTaskQueue.close()
```

Comment:
If you have tasks on the queue they could run at this point (between `roomJoinedLatch.countDown()` and `xmppTaskQueue.close()`.). Is that a problem?